lib/jekyll/page.rb

```diff
@@ -1,16 +1,16 @@
 module Jekyll
-  class Page
-    include Convertible
-
+  class Page < Document
     attr_writer :dir
-    attr_accessor :site, :pager
+    attr_accessor :pager
     attr_accessor :name, :ext, :basename
-    attr_accessor :data, :content, :output
+    attr_writer :data
 
     alias_method :extname, :ext
 
     FORWARD_SLASH = '/'.freeze
 
+    # Compatibility
```

Comment:
* Please use: `NOTE: COMPATIBILITY`, this makes it easy for people to find a list of what you consider "compatible"